week3/homework/src/actions/markAsDone.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+async function markAsDone(todo, request, response) {
+  try {
+    const todoId = request.params.id;
+
+    if (request.method === 'POST') {
```

Comment:
you already know the method when you define the operation. Perhaps it makes more sense to determine true or false there instead of here?